docker-images/kafka/scripts/kafka_tls_prepare_certificates.sh

```diff
@@ -72,19 +72,31 @@ create_keystore /tmp/kafka/cluster.keystore.p12 "$CERTS_STORE_PASSWORD" \
     "$HOSTNAME"
 echo "Preparing keystore for replication and clienttls listener is complete"
 
-CUSTOM_CERT_DIR="/opt/kafka/certificates/custom-tls-9093-certs"
-if [ -d "$CUSTOM_CERT_DIR" ]; then
-    echo "Preparing custom keystore for tls listener"
-    create_keystore_without_ca_file /tmp/kafka/custom-tls-9093.keystore.p12 "$CERTS_STORE_PASSWORD" "${CUSTOM_CERT_DIR}/tls.crt" "${CUSTOM_CERT_DIR}/tls.key" custom-key
-    echo "Preparing custom keystore for tls listener is complete"
-fi
+regex="^\/opt\/kafka\/certificates\/custom-([a-z0-9]{1,11})-([0-9]+)-certs$"
```

Comment:
Do we really need the regex? Is the prefix not enough? This is something what just ties it too much to changes in other places.

Reply:
Done.

Reply:
I do not think you addresses my main point. I think having the regex too exact is a risk because if we change for whatever reason the rules for port names we would need to do it on multiple places. So I wonder if using something more general for example such as `^\/opt\/kafka\/certificates\/(custom|oauth)-(.*+)-(.*+)-certs$` wouldn't work equally well with less dependency on the Java code.

Reply:
> wouldn't work equally well with less dependency on the Java code

So I am confused, is the above statement missing a question mark? Are you suggesting to use the more general grouping or not?

Reply:
Yeah, there is missing question mark, sorry ... I think more general would be better.